Documentation/operations/upgrade.rst

```diff
@@ -413,958 +362,6 @@ Helm Options
 * ``ipam.operator.clusterPoolIPv4PodCIDR`` was deprecated in favor of ``ipam.operator.clusterPoolIPv4PodCIDRList``
 * ``ipam.operator.clusterPoolIPv6PodCIDR`` was deprecated in favor of ``ipam.operator.clusterPoolIPv6PodCIDRList``
 
-.. _1.10_upgrade_notes:
```

Comment:
Similar to Quentin's feedback, I think it'd be a nice touch to Add a link to the previous branch's upgrade docs somewhere above here, maybe where we mention that users should upgrade to each minor release one at a time.